conda-installation.md

```diff
@@ -247,3 +260,66 @@ arguments = myscript.py
 transfer_input_files = Miniconda3-latest-Linux-x86_64.sh, script.py, other_input.file
 ```
 {:.sub}
+
+# Pinning Dependencies
```

Comment:
"Pinning" feels like jargon to me. :P Does something like "Specifying Dependencies" make more sense?